app/controllers/contents_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ContentsController < ApplicationController
+  layout 'dashboard'
+  before_action :authenticate_user!
+
+  def add_content
+    @content = Content.new({})
```

Comment:
why are we passing {} inside .new?

Reply:
I resolved this bhaiya. 
Thank you!

Reply:
@sks-wtag can you please check if your local changes are reflecting here or not?

Reply:
I resolved it 
Thank you.